node/resolvers/search/product.ts

```diff
@@ -165,7 +157,48 @@ export const resolvers = {
 
     recommendations: (product: SearchProduct) => product,
 
-    titleTag: prop('productTitle'),
+    description: ({ productId, description }: SearchProduct) => formatTranslatableStringV2({
+      content: description,
+      context: productId,
+    }),
+
+    metaTagDescription: ({ productId, metaTagDescription }: SearchProduct) => formatTranslatableStringV2({
+      content: metaTagDescription,
+      context: productId,
+    }),
+
+    titleTag: ({ productId, productTitle }: SearchProduct) => formatTranslatableStringV2({
+      content: productTitle,
+      context: productId,
+    }),
+
+    productName: ({ productId, productName }: SearchProduct) => formatTranslatableStringV2({
+      content: productName,
+      context: productId,
+    }),
+
+    linkText: async ({ productId, linkText }: SearchProduct, _: unknown, ctx: Context) => {
```

Comment:
Shouldn't this field be marked with `@translatableV2` directive at the schema?

Reply:
No, `translatableV2` translates to the user's current locale. We want to translate the linkText to the users binding

Reply:
👍